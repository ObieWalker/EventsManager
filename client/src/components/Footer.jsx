import React from 'react';
// import { Link } from 'react-router-dom';

const Footer = () => (
  <footer className="page-footer indigo index-footer footer-padding">
    <div >
      <div className="row">
        <div className="col l12 s12">
          <h5 className="white-text"> The Events Manager</h5>
          <p className="grey-text text-lighten-4">Â© Obi Walker (Andela 2017)</p>
        </div>
        <div className="col l4 offset-l2 s12">
        </div>
      </div>
      <div />
      <br />
      <br />
      <div className="row" />
    </div>
  </footer>
);
export default Footer;
