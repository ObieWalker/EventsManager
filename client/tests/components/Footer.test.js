import React from 'react';
import expect from 'expect';
import { shallow } from 'enzyme';
import Footer from '../../src/components/Footer.jsx';

let mountedForm;
let props;
let wrapper;

const getComponent = () => {
  if (!mountedForm) {
    props = {};
    mountedForm = shallow(<Footer {...props} />);
  }
  return mountedForm;
};

describe('Footer Component', () => {
  beforeEach(() => {});

  it('renders component successfully', () => {
    wrapper = getComponent();
    expect(wrapper).toMatchSnapshot();
  });
  it('renders component successfully', () => {
    wrapper = getComponent();
    expect(wrapper.find('footer').length).toBe(1);
    expect(wrapper.find('div').length).toBe(6);
    expect(wrapper.find('p').length).toBe(1);
    expect(wrapper.find('h5').length).toBe(1);
  });
});
