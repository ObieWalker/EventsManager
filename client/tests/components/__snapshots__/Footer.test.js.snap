// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component renders component successfully 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Footer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="container"
      >
        <div
          className="row"
        >
          <div
            className="col l6 s12"
          >
            <h5
              className="white-text"
            >
               The Events Manager
            </h5>
            <p
              className="grey-text text-lighten-4"
            >
              © Obi Walker (Andela 2017)
            </p>
          </div>
          <div
            className="col l4 offset-l2 s12"
          >
            <h5
              className="white-text"
            >
              Links
            </h5>
            <ul>
              <li>
                <a
                  className="white-text"
                  href="all-events.html"
                >
                  All Events
                </a>
              </li>
              <li>
                <a
                  className="white-text"
                  href="all-centers.html"
                >
                  All Centers
                </a>
              </li>
            </ul>
          </div>
        </div>
      </div>,
      "className": "page-footer indigo index-footer footer-padding",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="row"
        >
          <div
            className="col l6 s12"
          >
            <h5
              className="white-text"
            >
               The Events Manager
            </h5>
            <p
              className="grey-text text-lighten-4"
            >
              © Obi Walker (Andela 2017)
            </p>
          </div>
          <div
            className="col l4 offset-l2 s12"
          >
            <h5
              className="white-text"
            >
              Links
            </h5>
            <ul>
              <li>
                <a
                  className="white-text"
                  href="all-events.html"
                >
                  All Events
                </a>
              </li>
              <li>
                <a
                  className="white-text"
                  href="all-centers.html"
                >
                  All Centers
                </a>
              </li>
            </ul>
          </div>
        </div>,
        "className": "container",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="col l6 s12"
            >
              <h5
                className="white-text"
              >
                 The Events Manager
              </h5>
              <p
                className="grey-text text-lighten-4"
              >
                © Obi Walker (Andela 2017)
              </p>
            </div>,
            <div
              className="col l4 offset-l2 s12"
            >
              <h5
                className="white-text"
              >
                Links
              </h5>
              <ul>
                <li>
                  <a
                    className="white-text"
                    href="all-events.html"
                  >
                    All Events
                  </a>
                </li>
                <li>
                  <a
                    className="white-text"
                    href="all-centers.html"
                  >
                    All Centers
                  </a>
                </li>
              </ul>
            </div>,
          ],
          "className": "row",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h5
                  className="white-text"
                >
                   The Events Manager
                </h5>,
                <p
                  className="grey-text text-lighten-4"
                >
                  © Obi Walker (Andela 2017)
                </p>,
              ],
              "className": "col l6 s12",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": " The Events Manager",
                  "className": "white-text",
                },
                "ref": null,
                "rendered": " The Events Manager",
                "type": "h5",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "© Obi Walker (Andela 2017)",
                  "className": "grey-text text-lighten-4",
                },
                "ref": null,
                "rendered": "© Obi Walker (Andela 2017)",
                "type": "p",
              },
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h5
                  className="white-text"
                >
                  Links
                </h5>,
                <ul>
                  <li>
                    <a
                      className="white-text"
                      href="all-events.html"
                    >
                      All Events
                    </a>
                  </li>
                  <li>
                    <a
                      className="white-text"
                      href="all-centers.html"
                    >
                      All Centers
                    </a>
                  </li>
                </ul>,
              ],
              "className": "col l4 offset-l2 s12",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Links",
                  "className": "white-text",
                },
                "ref": null,
                "rendered": "Links",
                "type": "h5",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <li>
                      <a
                        className="white-text"
                        href="all-events.html"
                      >
                        All Events
                      </a>
                    </li>,
                    <li>
                      <a
                        className="white-text"
                        href="all-centers.html"
                      >
                        All Centers
                      </a>
                    </li>,
                  ],
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": <a
                        className="white-text"
                        href="all-events.html"
                      >
                        All Events
                      </a>,
                    },
                    "ref": null,
                    "rendered": Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "All Events",
                        "className": "white-text",
                        "href": "all-events.html",
                      },
                      "ref": null,
                      "rendered": "All Events",
                      "type": "a",
                    },
                    "type": "li",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": <a
                        className="white-text"
                        href="all-centers.html"
                      >
                        All Centers
                      </a>,
                    },
                    "ref": null,
                    "rendered": Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "All Centers",
                        "className": "white-text",
                        "href": "all-centers.html",
                      },
                      "ref": null,
                      "rendered": "All Centers",
                      "type": "a",
                    },
                    "type": "li",
                  },
                ],
                "type": "ul",
              },
            ],
            "type": "div",
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
    "type": "footer",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="container"
        >
          <div
            className="row"
          >
            <div
              className="col l6 s12"
            >
              <h5
                className="white-text"
              >
                 The Events Manager
              </h5>
              <p
                className="grey-text text-lighten-4"
              >
                © Obi Walker (Andela 2017)
              </p>
            </div>
            <div
              className="col l4 offset-l2 s12"
            >
              <h5
                className="white-text"
              >
                Links
              </h5>
              <ul>
                <li>
                  <a
                    className="white-text"
                    href="all-events.html"
                  >
                    All Events
                  </a>
                </li>
                <li>
                  <a
                    className="white-text"
                    href="all-centers.html"
                  >
                    All Centers
                  </a>
                </li>
              </ul>
            </div>
          </div>
        </div>,
        "className": "page-footer indigo index-footer footer-padding",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="row"
          >
            <div
              className="col l6 s12"
            >
              <h5
                className="white-text"
              >
                 The Events Manager
              </h5>
              <p
                className="grey-text text-lighten-4"
              >
                © Obi Walker (Andela 2017)
              </p>
            </div>
            <div
              className="col l4 offset-l2 s12"
            >
              <h5
                className="white-text"
              >
                Links
              </h5>
              <ul>
                <li>
                  <a
                    className="white-text"
                    href="all-events.html"
                  >
                    All Events
                  </a>
                </li>
                <li>
                  <a
                    className="white-text"
                    href="all-centers.html"
                  >
                    All Centers
                  </a>
                </li>
              </ul>
            </div>
          </div>,
          "className": "container",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="col l6 s12"
              >
                <h5
                  className="white-text"
                >
                   The Events Manager
                </h5>
                <p
                  className="grey-text text-lighten-4"
                >
                  © Obi Walker (Andela 2017)
                </p>
              </div>,
              <div
                className="col l4 offset-l2 s12"
              >
                <h5
                  className="white-text"
                >
                  Links
                </h5>
                <ul>
                  <li>
                    <a
                      className="white-text"
                      href="all-events.html"
                    >
                      All Events
                    </a>
                  </li>
                  <li>
                    <a
                      className="white-text"
                      href="all-centers.html"
                    >
                      All Centers
                    </a>
                  </li>
                </ul>
              </div>,
            ],
            "className": "row",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h5
                    className="white-text"
                  >
                     The Events Manager
                  </h5>,
                  <p
                    className="grey-text text-lighten-4"
                  >
                    © Obi Walker (Andela 2017)
                  </p>,
                ],
                "className": "col l6 s12",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": " The Events Manager",
                    "className": "white-text",
                  },
                  "ref": null,
                  "rendered": " The Events Manager",
                  "type": "h5",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "© Obi Walker (Andela 2017)",
                    "className": "grey-text text-lighten-4",
                  },
                  "ref": null,
                  "rendered": "© Obi Walker (Andela 2017)",
                  "type": "p",
                },
              ],
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h5
                    className="white-text"
                  >
                    Links
                  </h5>,
                  <ul>
                    <li>
                      <a
                        className="white-text"
                        href="all-events.html"
                      >
                        All Events
                      </a>
                    </li>
                    <li>
                      <a
                        className="white-text"
                        href="all-centers.html"
                      >
                        All Centers
                      </a>
                    </li>
                  </ul>,
                ],
                "className": "col l4 offset-l2 s12",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Links",
                    "className": "white-text",
                  },
                  "ref": null,
                  "rendered": "Links",
                  "type": "h5",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      <li>
                        <a
                          className="white-text"
                          href="all-events.html"
                        >
                          All Events
                        </a>
                      </li>,
                      <li>
                        <a
                          className="white-text"
                          href="all-centers.html"
                        >
                          All Centers
                        </a>
                      </li>,
                    ],
                  },
                  "ref": null,
                  "rendered": Array [
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": <a
                          className="white-text"
                          href="all-events.html"
                        >
                          All Events
                        </a>,
                      },
                      "ref": null,
                      "rendered": Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {
                          "children": "All Events",
                          "className": "white-text",
                          "href": "all-events.html",
                        },
                        "ref": null,
                        "rendered": "All Events",
                        "type": "a",
                      },
                      "type": "li",
                    },
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": <a
                          className="white-text"
                          href="all-centers.html"
                        >
                          All Centers
                        </a>,
                      },
                      "ref": null,
                      "rendered": Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {
                          "children": "All Centers",
                          "className": "white-text",
                          "href": "all-centers.html",
                        },
                        "ref": null,
                        "rendered": "All Centers",
                        "type": "a",
                      },
                      "type": "li",
                    },
                  ],
                  "type": "ul",
                },
              ],
              "type": "div",
            },
          ],
          "type": "div",
        },
        "type": "div",
      },
      "type": "footer",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
